CMakeLists.txt

```diff
@@ -228,6 +238,10 @@ if(NOT FORMAT_ONLY)
 
   include(library)
 
+  if(MSVC)
+    set(TPL_OPTIONS STATIC)
```

Comment:
Is this selecting a static version of some other library, or requesting that FleCSI itself be built as a static library?

Reply:
I have stopped using MSVC for working with FleCSI. If you like I can remove all of it being mentioned in the source tree again. 

For this concrete change, the intention was to force the FleCSI library to be built as a static library.

Reply:
If you think anyone might reasonably want to use it, I don't mind support for MSVC staying in (especially now that we have things like `NOMINMAX`).

I'm confused that an option named `TPL_OPTIONS` controls FleCSI itself rather than, well, the third-party libraries, but our build system is somewhat arcane.